amex-page.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>AMEX</title>
+        <style>
+            .container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            .column {
+                width: 25%;
+                float: left;
+            }
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            .col-2 {
+                width: 10%;
+                float: left;
+            }
+            .hero-image {
+                background-image: url(fistbump-over-desk.jpg);
+                background-position: center;
+                background-repeat: no repeat;
+                background-size: cover;
+                position: relative;
+                height: 50%;
+            }
+            .hero-text {
+                text-align: left;
+                position: absolute;
+                top: 30%;
+                left: 30%;
+                text-align: left;
+                color: black;
+            }
+            .credit-card {
+                width: 25%;
+                display: inline; 
+                float: left;               
+            }
+            .more-info {
+                margin-top: 30px;
+                width: 33%;
+                float: left;
+                display: inline;
+            }
+            .get-more {
+                margin-top: 50px;
+                width: 33%;
+                display: inline;
+                float: left;
+            }
+            h2 {
+                text-align: center;
+            }
+            .bb-footer {
+                margin-top: 50px;
+                float: left;
+                display: inline;
+            }
+            .col-footer {
+                width: 25%;
+                float: left;
+            }
+            .bb-footer li {
+                display: block;
+            }                   
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>    
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="col-2">
```

Comment:
col-2 does not make sense here, as there aren't any other columns in this row.
This should just be column, for now